### DISCUSSION TITLE
Can I use transactions in Model.Create()?

### DISCUSSION THREAD

**Discussion by fnoquiq** [adonisjs/core#1732] [Help] [answered]
How can I use transactions in Model.Create method?

```ts
await User.create({...userParams}, trx(?))
```

Or I can't?

**Comment by thetutlage** [comment, marked as answer]
Which version of Lucid are you using?

**Reply by fnoquiq** [reply]
@adonisjs/lucid: 8.4.2

**Reply by thetutlage** [reply]
You can use it as follows

```ts
User.create({...userParams}, { client: trx })
```

Also, I recommend using VsCode or an editor that has Intellisense for Typescript. It will help you a lot

**Reply by fnoquiq** [reply]
Thanks for the help, I didn't find it in the documentation.

---

> Also, I recommend using VsCode or an editor that has Intellisense for Typescript. It will help you a lot

I'm using VSCode, but I got lost while browsing Lucid's interfaces. My first contact with typescript was a few months ago.
